torchprime/experimental/torchax_models/DeepSeek-V3/inference/requirements.txt

```diff
@@ -0,0 +1,4 @@
+torch==2.4.1
```

Comment:
We should merge these into https://github.com/AI-Hypercomputer/torchprime/blob/main/pyproject.toml#L15 so we don't have a separate set of deps.